core/src/main/java/org/infinispan/xsite/irac/NoOpIracManager.java

```diff
@@ -23,6 +24,9 @@
 
    private static final NoOpIracManager INSTANCE = new NoOpIracManager();
 
+   private final LongAdder conflictsCounts = new LongAdder();
```

Comment:
can be removed. `NoOpIracManager` is used when IRAC is disabled. So these counters will never be updated.

Reply:
Push the changes, change all methods to no-ops